src/Sylius/Bundle/AttributeBundle/Resources/config/validation.xml

```diff
@@ -36,34 +41,31 @@
     </class>
 
     <class name="Sylius\Component\Attribute\Model\AttributeTranslation">
-        <property name="presentation">
+        <property name="name">
             <constraint name="NotBlank">
-                <option name="message">sylius.attribute.presentation.not_blank</option>
+                <option name="message">sylius.attribute.name.not_blank</option>
                 <option name="groups">sylius</option>
             </constraint>
             <constraint name="Length">
                 <option name="min">2</option>
-                <option name="minMessage">sylius.attribute.presentation.min_length</option>
+                <option name="minMessage">sylius.attribute.name.min_length</option>
                 <option name="max">255</option>
-                <option name="maxMessage">sylius.attribute.presentation.max_length</option>
+                <option name="maxMessage">sylius.attribute.name.max_length</option>
                 <option name="groups">sylius</option>
             </constraint>
         </property>
     </class>
 
     <class name="Sylius\Component\Attribute\Model\AttributeValue">
+        <constraint name="Sylius\Bundle\AttributeBundle\Validator\Constraints\AttributeValue">
```

Comment:
``ValidAttributeValue`` I know it sounds kinda odd but it follow the convention from core constraints. AttributeValue alone does not have any meaning in that context.